docs/extension-maintainers.md

```diff
@@ -160,6 +160,26 @@ should specify this path in `build-path`, for example:
 }
 ```
 
+The `build-path` may contain some templated values which are replaced:
+
+ * `{version}` to be replaced with the package version. For example a package
+   with version 1.2.3 with a `build-path` of `myext-{version}` the actual build
+   path would become `myext-{version}`.
+
+#### `override-download-url-method`
+
+The `override-download-url-method` directive allows extension maintainers to
```

Comment:
should it have `override` in the name ?

Reply:
I figured it might be better, since you're overriding the default behaviour (i.e. use what Composer does). If you think it's superfluous, I'm happy to remove it? :thinking:

Reply:
Well, *any* configuration option overrides the default behavior when you specify a value. That's the reason to make it configurable.

Reply:
Upstream PR sent - thank you composer/composer#12297